txrelayer/txrelayer.go

```diff
@@ -130,6 +135,12 @@ func (t *TxRelayerImpl) sendTransactionLocked(txn *ethgo.Transaction, key ethgo.
 		return ethgo.ZeroHash, err
 	}
 
+	if t.writer != nil {
+		_, _ = t.writer.Write([]byte(
+			fmt.Sprintf("[ROOTCHAIN - CONTRACTS DEPLOYMENT TRANSACTION DETAIL]\nFrom = %s \nGas = %d  \nGas Price = %d\n",
```

Comment:
@Nemanja0x would we like to log Input field as well?

Reply:
Ok, then add that as well.

Reply:
We already have additional information. Log currently looks like this 

INFO [06-19|10:48:59.069] Submitted contract creation
hash=0xdf1ee50904451a1dcd7d75a6e5c1e4725856a0792d56630c4a8d49a5e5258c8e
from=0x1FC1411A3Bd09E63d4A306FD1EB838bA457ddA13 nonce=3
contract=0x436604426F31A05f905C64edc973E575BdB46471 value=0
[TxRelayer.SendTransaction]
From = 0x1FC1411A3Bd09E63d4A306FD1EB838bA457ddA13 
Gas = 5242880 
Gas Price = 477436156

And also later in log 

[ROOTCHAIN - DEPLOY CONTRACT]
Name                   = RootERC721Predicate
Contract (address)     = 0x436604426F31A05f905C64edc973E575BdB46471
Transaction (hash)     = 0xdf1ee50904451a1dcd7d75a6e5c1e4725856a0792d56630c4a8d49a5e5258c8e
Transaction (gas used) = 1458136